addons/binding/org.openhab.binding.mihome/ESH-INF/thing/aqara_86sw1.xml

```diff
@@ -0,0 +1,28 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<thing:thing-descriptions bindingId="mihome" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+	xmlns:thing="http://eclipse.org/smarthome/schemas/thing-description/v1.0.0"
+	xsi:schemaLocation="http://eclipse.org/smarthome/schemas/thing-description/v1.0.0 http://eclipse.org/smarthome/schemas/thing-description-1.0.0.xsd">
+
+	<thing-type id="86sw1">
+		<supported-bridge-type-refs>
+			<bridge-type-ref id="bridge" />
+		</supported-bridge-type-refs>
+
+		<label>Xiaomi Aqara 1 Channel Smart Light Control</label>
+		<description>Battery powered Aqara Switch with 1 Channel</description>
+
+		<channels>
+			<channel id="ch1" typeId="sw1-trg" />
+			<channel id="batteryLevel" typeId="system.battery-level" />
+			<channel id="lowBattery" typeId="system.low-battery" />
+		</channels>
+
+		<config-description>
+			<parameter name="itemId" type="text" readOnly="true">
+				<label>Item ID</label>
+				<description>An identifier for the item connected.</description>
```

Comment:
Forget about what?
From your thing file example, it seems that this is a mandatory parameter that the user has to provide - so it should not be advanced and readOnly doesn't make any sense either.
Btw, it would be nice if you could rework the README.md to be closer to our [suggested template](https://github.com/eclipse/smarthome/blob/master/tools/archetype/binding/src/main/resources/archetype-resources/README.md), i.e. it should clearly describe the configuration parameters that are required (and thus the Item Id as well, which isn't mentioned yet at all).